kuttl/tests/release-1.1.0/upgrade-to-1.1.0.sh

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env bash
```

Comment:
This seems a replica of tools/upgrade-to-1.1.0.sh . Can't the kuttl test cases reuse the script present there? Any changes to the original script will need to reflect here too

Reply:
I am thinking if I need to use links to the files. will do some research today to find if I can re-use.